monitoring/jaeger-mixin/README.md

```diff
@@ -0,0 +1,82 @@
+# Prometheus monitoring mixin for Jaeger
```

Comment:
Should this be mentioned (linked to) in the main readme?

Reply:
Not right now, I'd like to get one round of feedback first.